### PR TITLE
Update version to 0.75.0

### PR DESCRIPTION
**Pull request by chvck**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "couchbase-shell"
-version = "1.0.0-beta.5"
+version = "0.75.0"
 authors = ["Michael Nitschinger <michael@nitschinger.at>", "Charles Dixon <chvckd@gmail.com>"]
```
Comment (ingenthr) [inline Cargo.toml:4]: Should we update this and the next line?

**Review by ingenthr — APPROVED** [review]
Might want to update some of the other package metadata.

File: Cargo.toml (hunk 1 above)
Comment (chvck) [reply Cargo.toml:4]: Updating the edition should be its own commit, it'll require a bunch of testing etc... as it's referring to the underlying rust language compatibility and could break some things that will need fixing.

Comment (ingenthr) [reply Cargo.toml]: ah, ok.  that's fine then.

**Pull request merged**